發生時間：2014-07-02 10:51:20.0133
類別：AppException
訊息：已發生類型 'System.Web.HttpUnhandledException' 的例外狀況。
   於 System.Web.UI.Page.HandleError(Exception e)
   於 System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
   於 System.Web.UI.Page.ProcessRequest(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
   於 System.Web.UI.Page.ProcessRequest()
   於 System.Web.UI.Page.ProcessRequestWithNoAssert(HttpContext context)
   於 System.Web.UI.Page.ProcessRequest(HttpContext context)
   於 ASP.mat_mat2_mat2108_02_aspx.ProcessRequest(HttpContext context)
   於 System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()
   於 System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously)
方法：[Void Invoke(System.Object, System.EventArgs)] => [Void Application_Error(System.Object, System.EventArgs)] => [Void WriteErrorLog(System.String, System.String)]
例外堆疊：
=============================================================
發生時間：2014-07-02 10:51:43.0656
類別：AppException
訊息：已發生類型 'System.Web.HttpUnhandledException' 的例外狀況。
   於 System.Web.UI.Page.HandleError(Exception e)
   於 System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
   於 System.Web.UI.Page.ProcessRequest(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
   於 System.Web.UI.Page.ProcessRequest()
   於 System.Web.UI.Page.ProcessRequestWithNoAssert(HttpContext context)
   於 System.Web.UI.Page.ProcessRequest(HttpContext context)
   於 ASP.mat_mat2_mat2108_02_aspx.ProcessRequest(HttpContext context)
   於 System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()
   於 System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously)
方法：[Void Invoke(System.Object, System.EventArgs)] => [Void Application_Error(System.Object, System.EventArgs)] => [Void WriteErrorLog(System.String, System.String)]
例外堆疊：
=============================================================
發生時間：2014-07-02 10:55:16.1402
類別：AppException
訊息：已發生類型 'System.Web.HttpUnhandledException' 的例外狀況。
   於 System.Web.UI.Page.HandleError(Exception e)
   於 System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
   於 System.Web.UI.Page.ProcessRequest(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
   於 System.Web.UI.Page.ProcessRequest()
   於 System.Web.UI.Page.ProcessRequestWithNoAssert(HttpContext context)
   於 System.Web.UI.Page.ProcessRequest(HttpContext context)
   於 ASP.mat_mat2_mat2108_02_aspx.ProcessRequest(HttpContext context)
   於 System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()
   於 System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously)
方法：[Void Invoke(System.Object, System.EventArgs)] => [Void Application_Error(System.Object, System.EventArgs)] => [Void WriteErrorLog(System.String, System.String)]
例外堆疊：
=============================================================
發生時間：2014-07-02 10:55:54.8690
類別：AppException
訊息：已發生類型 'System.Web.HttpUnhandledException' 的例外狀況。
   於 System.Web.UI.Page.HandleError(Exception e)
   於 System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
   於 System.Web.UI.Page.ProcessRequest(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
   於 System.Web.UI.Page.ProcessRequest()
   於 System.Web.UI.Page.ProcessRequestWithNoAssert(HttpContext context)
   於 System.Web.UI.Page.ProcessRequest(HttpContext context)
   於 ASP.mat_mat2_mat2108_02_aspx.ProcessRequest(HttpContext context)
   於 System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()
   於 System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously)
方法：[Void Invoke(System.Object, System.EventArgs)] => [Void Application_Error(System.Object, System.EventArgs)] => [Void WriteErrorLog(System.String, System.String)]
例外堆疊：
=============================================================
